oldabe/money_in.py

```diff
@@ -504,22 +519,26 @@ def get_amounts_owed(total_amount, attributions):
             for email, share in attributions.items()}
 
 
-def distribute_remaining_amount(remaining_amount, truncated_payable_attributions, payment):
+def redistribute_pot(redistribution_pot, attributions, unpayable_contributors, payment_file, amounts_owed):
```

Comment:
Maybe we should rename `amounts_owed` -> `amounts_payable` since it's how much we're paying out, but may be more than contributors are "owed" according to attributions?

Reply:
Yeah I think that makes more sense 👍